packages/wallet/customEnvironment.ts

```diff
@@ -0,0 +1,13 @@
+const Environment = require('jest-environment-jsdom');
+
+// Cbor package requires a textEncoder to hav been defined
+module.exports = class CustomTestEnvironment extends Environment {
```

Comment:
Aren't we using CSL to decode cbor?
If it's really required, I think it's better to put it under `/test` and document required polyfill in README.

Reply:
There may have been early discussion around not using the CSL to stay a bit more generic, but with it we can make better assertions in addition to avoiding this extra test polyfill